components/BottomNavigation/examples/MDCBottomNavigationBarController.m

```diff
@@ -0,0 +1,278 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCBottomNavigationBarController.h"
+
+@interface MDCBottomNavigationBarController ()
+
+/** The view that hosts the content for the selected view controller **/
+@property(nonatomic, strong) UIView *content;
+
+/** Constrains the navigation bar's height to the constant value of this property **/
+@property(nonatomic, strong) NSLayoutConstraint *navigationBarHeightConstraint;
+
+@end
+
+@implementation MDCBottomNavigationBarController
+
+- (instancetype)init {
+  self = [super init];
+  if (self) {
+    _navigationBar = [[MDCBottomNavigationBar alloc] init];
+    _content = [[UIView alloc] init];
+    _viewControllers = @[];
+    _selectedIndex = NSNotFound;
+  }
+
+  return self;
+}
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+
+  _navigationBar.delegate = self;
```

Comment:
Please use `self.navigationBar` outside of the initializer.  Same for the other properties where it makes sense.